### PR TITLE
#65 admin/promo/create

### PR DESCRIPTION
**Pull request by twocs**
### Description of the Change

Implemented an endpoint (/api/admin/promo/create) for an admin to create a promo for a note.

### Issue or RFC

Issue #65 
Documentation: https://misskey.io/api-doc#tag/admin

### Verification Process

Unit test for creating a promo.

### Release Notes

Created an endpoint for admin to promo a note.

### CODE REVIEW

**Comment by twocs** [comment]
@yitsushi I'm participating in Hacktoberfest. If this is approved please confirm whether I can also create endpoints for other issues, such as #44, #20. #17, #16, #13, #68, or #7.

File: services/admin/promo/create_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		[]core.BaseRequest{
+			promo.CreateRequest{},
+		},
+		[]core.BaseRequest{
+			promo.CreateRequest{NoteID: "8zwxx3cpy7", ExpiresAt: 0},
+			promo.CreateRequest{NoteID: "8zwxx3cpy8", ExpiresAt: 0},
+			promo.CreateRequest{NoteID: "8zwxx3cpy9", ExpiresAt: time.Now().Add(86400 * 24 * time.Hour).Unix()},
+		},
+	)
+}
```
Comment (yitsushi) [inline services/admin/promo/create_test.go:67]: Can you please add an example in the test, like other [endpoints has](https://github.com/yitsushi/go-misskey/blob/74f1c29bd87879642f3a75d429506a2d5e65126f/services/promo/mark_as_read_test.go#L29)? That way we always have simple examples for all endpoints that can be copy pasted if someone wants to play around.

**Review by yitsushi — CHANGES_REQUESTED** [review]
Please add an `ExampleService_xxxxxxx`.

You can add as many endpoints, or fix existing ones as you want 🤣 you can break it down one PR per endpoints to make it clean.

**Comment by twocs** [comment]
Have updated to fix the golint and golang-ci-lint issues.

**Comment by twocs** [comment]
Have also added an example of creating a promo request.

File: services/admin/promo/create_test.go (hunk 1 above)
Comment (twocs) [reply services/admin/promo/create_test.go:64]: Have added an example of creating a promo request, following the example.

**Review by yitsushi — APPROVED** [review]

File: services/admin/promo/create_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+func ExampleService_CreatePromo() {
```
Comment (yitsushi) [inline services/admin/promo/create_test.go:66]: The name of the example has to match the name of the function it demonstrates, that's how Go connects functions with their examples.

```suggestion
func ExampleService_Create() {
```

**Pull request merged**